administrator/components/com_actionlogs/Field/PlugininfoField.php

```diff
@@ -19,7 +22,7 @@
  *
  * @since  3.9.2
  */
-class JFormFieldPluginInfo extends JFormField
+class PlugininfoField extends FormField
```

Comment:
```suggestion
class PluginInfoField extends FormField
```

Might be wrong on this one but it reads better

Reply:
This would also require renaming the file to `PluginInfoField.php` and updating XML forms which use the field.

Reply:
really? the class was camelcase before

Reply:
Classes are case insensitive in PHP so any casing worked in J3. But with namespaced classes this is required for autoloading. Calling `Joomla\Component\Actionlogs\Administrator\Field\PluginInfoField` would try to load `administrator/components/com_actionlogs/Field/PluginInfoField.php` which would fail on systems with case-sensitive filesystems.

Reply:
If we are going to change this one, we should change them in all components. That is for another PR.